ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,19 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-tolatest-f28
```

Comment:
From the test result, the multi build is used. We need it for amd64, right?

```
INFO[2024-03-21T11:28:23Z] Requesting 4.14 from https://api.integration.openshift.com/api/upgrades_info/graph?arch=multi&channel=fast-4.14 
INFO[2024-03-21T11:28:23Z] Resolved release latest to quay.io/openshift-release-dev/ocp-release@sha256:89c56f1[7](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/47434/rehearse-47434-periodic-ci-openshift-verification-tests-master-ota-multi-stable-4.15-upgrade-from-stable-4.14-aws-ipi-tolatest-f28/1770774423276621824#1:build-log.txt%3A7)275cd59fcb77927d9eead809ee95957077992f57dd7e2a6e8b464c09 
INFO[2024-03-21T11:28:23Z] Requesting a release from https://multi.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable-multi/latest?in=%3E4.15.0-0+%3C4.16.0-0 
INFO[2024-03-21T11:28:23Z] Resolved release target to quay.io/openshift-release-dev/ocp-release:4.15.3-multi
```

Reply:
I think we are testing --to-latest option, so it don't care about architecture.

Reply:
yes, for the test it doesn't care about arch, but to be consistent with original test coverage in automation, i would suggest to run it on amd64. 

btw, per my understanding, the job name should be `amd-mixarch` or `arm-mixarch` if it's for a multi payload, but the job name here seems for amd64. i'm not expert on the job name rule, cc @jianlinliu to help confirm about mixarch job name.

Reply:
In this ci yaml, it is using multi-arch payload to run the OTA upgrade testing, but we do not have to add `mixarch`, multi-arch payload is supposed to work with single arch (whatever amd or arm) and multi-arch instances, so `mixarch` is not needed. But I agree with `to be consistent with original test coverage in automation`, so I suggest to add `amd` to the job name as `aws-ipi-amd-tolatest-f28`